### DISCUSSION TITLE
Support for URL Launch Scheme to open OsmApp with directions from another app

### DISCUSSION THREAD

**Discussion by dteirney** [osmandapp/OsmAnd#13597] [Q&A]
Hello,

Does OsmAnd app support URL launch schemes for iOS (and the similar concept for Android)? Would like to be able to launch OsmApp from a seperate app on the device to be able to show turn-by-turn directions from and to given coordinates with a specified vehicle profile.

Google Maps provides this for example using the native support provided by iOS. Details on https://developers.google.com/maps/documentation/urls/ios-urlscheme. Similar approach on Android using intents, https://developers.google.com/maps/documentation/urls/android-intents

Thanks!
David

**Comment by garrettboone** [comment]
https://osmand.net/go?lat=40.96866989135742&lon=-81.35277557373047

If you put this in a browser it will ask if you want to open it in the app. But for me on Android when I click this from anywhere else (like calendar), it prompts to open in the app where I can select Always. It's one of the formats used when exporting a location from the Details section of the location.

**Reply by garrettboone** [reply]
Or maybe even `geo:40.96866989135742,-81.35277557373047;u=15` for mobile devices.
